# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296356367 28800
#      Sat Jan 29 18:59:27 2011 -0800
# Node ID 61a4171c057f7c8c11826837abb087ce4fb978d4
# Parent  9611014e7cf1054849d841a3c834ab31d3d704b0
Add glpkmex function to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-29  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/optim.txi: Add glpkmex function to documentation.
+
 2011-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/system.txi: Add compare_versions function to 
 	documentation.
 
 2011-01-27  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/numbers.txi: Add sizemax function to documentation.
diff --git a/doc/interpreter/optim.txi b/doc/interpreter/optim.txi
--- a/doc/interpreter/optim.txi
+++ b/doc/interpreter/optim.txi
@@ -65,16 +65,18 @@ subject to the linear constraints
 @math{A*x = b} where @math{x @geq{} 0}.
 @end ifnottex
 
 @noindent
 The @code{glpk} function also supports variations of this problem.
 
 @DOCSTRING(glpk)
 
+@DOCSTRING(glpkmex)
+
 @node Quadratic Programming
 @section Quadratic Programming
 
 Octave can also solve Quadratic Programming problems, this is
 @tex
 $$
  \min_x {1 \over 2} x^T H x + x^T q
 $$
diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-29  Rik  <octave@nomad.inbox5.com>
+
+	* optimization/glpkmex.m: Add seealso reference to glpk.
+
 2011-01-29  Rik  <octave@nomad.inbox5.com>
 
 	* miscellaneous/compare_versions.m: Allow only "==" equality operator.
 	Add input validation tests.  Improve docstring.
 
 	* miscellaneous/ver.m, miscellaneous/version.m, path/matlabroot.m:
 	Improve docstring.
 
diff --git a/scripts/optimization/glpkmex.m b/scripts/optimization/glpkmex.m
--- a/scripts/optimization/glpkmex.m
+++ b/scripts/optimization/glpkmex.m
@@ -16,16 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {[@var{xopt}, @var{fmin}, @var{status}, @var{extra}] =} glpkmex (@var{sense}, @var{c}, @var{A}, @var{b}, @var{ctype}, @var{lb}, @var{ub}, @var{vartype}, @var{param}, @var{lpsolver}, @var{save_pb})
 ## This function is provided for compatibility with the old @sc{matlab}
 ## interface to the GNU @sc{glpk} library.  For Octave code, you should use
 ## the @code{glpk} function instead.
+## @seealso{glpk}
 ## @end deftypefn
 
 function [xopt, fopt, status, extra] = glpkmex (varargin)
 
   ## If there is no input output the version and syntax
   if (nargin < 4 || nargin > 11)
     print_usage ();
     return;
